server/BraveAlerterConfigurator.js

```diff
@@ -137,10 +152,14 @@ class BraveAlerterConfigurator {
     switch (fromAlertState) {
       case CHATBOT_STATE.STARTED:
       case CHATBOT_STATE.WAITING_FOR_REPLY:
-        returnMessage = t('incidentCategoryRequest', {
-          lng: language,
-          incidentCategories: this.createResponseStringFromIncidentCategories(incidentCategories, language),
-        })
+        if (toAlertState === CHATBOT_STATE.RESET) {
+          returnMessage = null
```

Comment:
To follow the general chatbot principle of always sending a reply, maybe we just want to send the "Thank you" text message and `logError` the fact that we're in a state that we shouldn't ever be in?

Reply:
Ah, this answers my above comment. I agree with you here.

Reply:
I've made it such that if the chatbot ever enters RESET (either X --> RESET, or RESET --> RESET), the chatbot simply replies 'Thank you.' This should be enough to acknowledge the client's messages without seeming broken. (This shouldn't ever happen anyways, as `getClientMessageForRequestToReset` returns null, but good to have a backup!)